console/src/app/modules/search-user-autocomplete/search-user-autocomplete.component.ts

```diff
@@ -26,6 +26,8 @@ export enum UserTarget {
   EXTERNAL = 'external',
 }
 
+const USER_LIMIT = 100;
```

Comment:
Why are 10 not enough?
Are 100 users cached?

Reply:
100 users for a selection does not make sense to me either

Reply:
lets do 25 😉